src/server/controllers/account-transactions.js

```diff
@@ -254,7 +269,9 @@ const defaultFields = [
   'netAmount',
   'currency',
   'accountSlug',
+  'accountName',
   'oppositeAccountSlug',
+  'oppositeAccountName',
```

Comment:
I'm wondering, should we keep slug if we decide to have name?

Reply:
Not a blocker but something to have in mind.

Reply:
@znarf `slug` is still interesting to quickly go from the CSV to the profile on Open Collective. Maybe we could replace `accountSlug` by `accountURL` and make its value `https://opencollective.com/${slug}`?